test/test_02_device.py

```diff
@@ -1,23 +1,33 @@
-import pytest
 from helper import Helper
 
-class TestUser:
 
-    def test_device_can_register(self):
+class TestDevice:
+    def test_device_can_upload_cptv(self):
         helper = Helper()
 
-        description = "If a new device 'The Destroyer' signs up"
-        destroyer = helper.given_new_device(self, 'The Destroyer', description=description)
-        
-        print("Then 'The Destroyer' should able to log in")
-        the_destroyer = helper.login_as_device(destroyer.devicename)
+        description = "If a new device 'Destroyer' signs up"
+        destroyer = helper.given_new_device(self, 'Destroyer', description=description)
 
-        print("And 'The Destroyer' should be able to upload a CPTV file")
-        recording = the_destroyer.upload_recording()
+        print("Then 'Destroyer' should able to log in")
+        helper.login_as_device(destroyer.devicename)
 
-        print("And the CPTV file should be visible to super users on the site")
-        helper.admin_user().can_see_recordings(recording)
+        print("And 'Destroyer' should be able to upload a CPTV file")
+        destroyer.upload_recording()
 
+        print("And the CPTV file should be visible to super users")
+        helper.admin_user().can_see_recording_from(destroyer)
 
+    def test_device_can_upload_audio(self):
```

Comment:
Not a biggy but I would be tempted to put it in a different file with a different filename.  Since one is a cell phone device and one is a thermal camera device.  Keep the tests based around user scenarios.

Reply:
Sounds good. I'll split them up.